front/app/containers/Admin/projects/edit/description/index.tsx

```diff
@@ -56,6 +56,8 @@ const ProjectDescription = memo<Props & InjectedIntlProps & WithRouterProps>(
       intl: { formatMessage },
     } = props;
 
+    const [contentBuilderModuleActive, setContentBuilderModuleActive] =
```

Comment:
Let's rename this to moduleActive to avoid referencing contentBuilder in our main app 🙏